README.md

```diff
@@ -48,19 +49,19 @@ you can omit this parameter.**
 
 Usage
 ---------------------
-To build simple [nodejs-echo-app](https://github.com/ryanj/node-echo) application,
+To build simple [nodejs-sample-app](https://github.com/openshift/sti-nodejs/tree/master/0.10/test/test-app) application,
 using standalone [STI](https://github.com/openshift/source-to-image) and then run the
 resulting image with [Docker](http://docker.io) execute:
 
 *  **For RHEL based image**
     ```
-    $ sti build https://github.com/ryanj/node-echo.git openshift/nodejs-010-rhel7 nodejs-echo-app
+    $ sti build https://github.com/openshift/sti-nodejs.git --contextDir=/0.10/test/test-app/ openshift/nodejs-010-rhel7 nodejs-sample-app
```

Comment:
With config you'll end up telling user to be in that and not in other directory, with this I don't care if he has or not the source code of this repo. It's more universal this way.